src/main/java/com/openshift/jenkins/plugins/pipeline/OcAction.java

```diff
@@ -46,20 +46,21 @@ public OcAction(String server, String project, String verb, List verbArgs, List
         this.reference = reference==null?(new HashMap<String,String>()):reference;
     }
 
-    public static void exitStatusRaceConditionBugWorkaround( Controller dtc, FilePath filePath, Launcher launcher ) throws InterruptedException, IOException {
+    public static Integer exitStatusRaceConditionBugWorkaround( Controller dtc, FilePath filePath, Launcher launcher ) throws InterruptedException, IOException {
         for ( int tries = 30; tries > 0; tries-- ) {
             try {
                 // exitStatus can throw an IOException (reporting a NumberFormatException) if the PID file has been created but not
-                // populated. Make sure it stops throwing this exception before continuing.
-                dtc.exitStatus(filePath,launcher);
-                return; // exitStatus is healthy, we should be safe
+                // populated, or if the PID file has not yet been created. Make sure it stops throwing this exception before continuing.
+                Integer i = dtc.exitStatus(filePath,launcher);
+                return i; // exitStatus is healthy, we should be safe
             } catch ( IOException ioe ) {
                 if ( tries == 1 ) {
                     throw ioe;
                 }
-                Thread.sleep(1000);
+                Thread.sleep(125);
             }
         }
+        return null;
```

Comment:
should log some details here when we give up, about what pid/file we were looking for and noting that we gave up on it.